Src/BootCamp.Chapter/Item.cs

```diff
@@ -15,6 +15,10 @@ public decimal GetPrice()
         }
 
         private float _weight;
+        public float GetWeight()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetWeight' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5148522)